### PR TITLE
Improve basetest

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

**Comment by mortenjc** [comment]
Removed UDP communications from CaenBaseTest as a, well, test. Instead we  are writing 
directly to the receive RingBuffer.
Hopefully this can improve the stability of the test. In any case it looks simpler now and enabled to
increase the test coverage by forcing a FIFO sequence error.

File: src/common/readout/ess/Parser.cpp [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 namespace ESSReadout {
 
-// #undef TRC_LEVEL
-// #define TRC_LEVEL TRC_L_WAR
+#undef TRC_LEVEL
+#define TRC_LEVEL TRC_L_WAR
```
Comment (tbukovics) [inline src/common/readout/ess/Parser.cpp:18]: Is this change for good reason or it left after test and debug?

**Review by tbukovics — APPROVED** [review]
I found one point where I believe some debug flags left on. Otherwise this seems a very good reasonable change and improvement on the test.

File: src/common/readout/ess/Parser.cpp (hunk 1 above)
Comment (mortenjc) [reply src/common/readout/ess/Parser.cpp:18]: Yes, it was on purpose, but I suppose that this is something we should discuss.

**Pull request merged**